code/script.js

```diff
@@ -2,6 +2,22 @@
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
+const findOutButton = document.getElementById('filter')
+const winOrLose = document.getElementById('winOrLose');
+const winOrLoseText = document.getElementById('winOrLoseText')
+const playAgainButton = document.getElementById('playAgain')
+const counterText = document.getElementById('counterText');
+const counter = document.getElementById('counter')
+const secretPersonName = document.getElementById('secretPersonName')
+const secretPersonImg = document.getElementById('secretPersonImg')
+const popUp = document.getElementById('popUp')
+const overlay = document.getElementById('overlay')
+const popUpButton = document.getElementById('popUpButton')
+const popUpMessage = document.getElementById('popUpMessage')
+const confirmMessage = document.getElementById('confirmMessage')
+const popUpConfirm = document.getElementById('popUpConfirm')
+const confirmYesButton = document.getElementById('confirmYesButton')
+const confirmNoButton = document.getElementById('confirmNoButton')
```

Comment:
It makes it much easier to follow your code later on with all these DOM selectors, thanks for that, Joanna!